### PR TITLE
R: updates for packages and a couple of fix-ups

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Updates and fixed.

_Nothing will be added here._

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`? (where added)
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: R/R-Bayesrel/Portfile [hunk 1]
```diff
@@ -14,15 +15,18 @@ checksums           rmd160  19360a42269db0b708e34e58b8673417c1f6ca0f \
                     sha256  ed41d6dcf3adf55fc65dad7aec91fec2dfedfc2340bcc0bbfd4d4728e9bcdb30 \
                     size    570104
 
+# https://github.com/juliuspfadt/Bayesrel/pull/27
```
Comment (ryandesign) [inline R/R-Bayesrel/Portfile:18]:
```suggestion
# https://github.com/juliuspfadt/Bayesrel/issues/28
```

File: R/R-Rcsdp/Portfile [hunk 2]
```diff
@@ -13,6 +14,9 @@ checksums           rmd160  04e56b4204a2dac2d134043cc3c7db3d5a4bc726 \
                     sha256  50048bcb4a8bb9f9b48c5e43a32126eda1d23d17876c7632e20b04953f3b1cd2 \
                     size    498160
 
+# https://github.com/juliuspfadt/Bayesrel/pull/27
```
Comment (ryandesign) [inline R/R-Rcsdp/Portfile:17]:
```suggestion
# https://github.com/juliuspfadt/Bayesrel/issues/28
```

File: R/R-blackbox/Portfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+checksums           rmd160  3d6e8d1f73a5f2b738e86df211086baa77f78d48 \
+                    sha256  14d1a501470b441f95e266f6716631010bd97213c0cdaed200ef9769cb1ea90e \
+                    size    185404
+
+depends_lib-append  port:R-foreach \
+                    port:R-geometry \
+                    port:R-matrixStats \
+                    port:R-nloptr \
+                    port:R-numDeriv \
+                    port:R-nloptr \
```
Comment (ryandesign) [inline R/R-blackbox/Portfile:22]: Specified twice, as pointed out by `port lint`.
```suggestion
```

**Comment by barracuda156** [comment]
Let me squash, rebase and add a few updates.

**Comment by barracuda156** [comment]
@ryandesign @herbygillot Should be good now.

**Comment by barracuda156** [comment]
@ryandesign @herbygillot Do we still have any issues with this one? I think I have addressed Ryan’s suggestion (they were merged and squashed).

**Comment by barracuda156** [comment]
@herbygillot @pmetzger Bump

**Comment by pmetzger** [comment]
I feel uncomfortable merging these mega-updates because I have trouble checking them over thoroughly. I'll look this time though.

**Comment by pmetzger** [comment]
I'm sort of relying on @ryandesign's review here.

**Pull request merged**